src/clients/AcrossConfigStoreClient.ts

```diff
@@ -27,9 +26,7 @@ import {
   DisabledChainsUpdate,
 } from "../interfaces";
 
-import * as lpFeeCalculator from "../lpFeeCalculator";
-import { BlockFinder, across } from "@uma/sdk";
-import { HubPoolClient } from "./HubPoolClient";
+import { across } from "@uma/sdk";
```

Comment:
Why do you import this?

Reply:
It's already imported (see line 31 above).